src/org-unit-tree/OrgUnitTree.component.js

```diff
@@ -100,6 +100,7 @@ class OrgUnitTree extends React.Component {
                         children: true,
                         path: true,
                         parent: true,
+                        geometry: true,
```

Comment:
This is has a side-effect for everyone using the component, geometry will be requested unconditionally. With MultiPolygon geometries, that may increase request sizes. To evaluate with your PM. Two approaches:

- [simple] Ask for field "geometry" only if onChildrenLoaded is requested (as the component itself does not need)
- [more complex] Change the prop so it works like this: `onChildrenLoaded={ {fields: "geometry", fn: someFuncion }}`

Reply:
I used the complex solution, it could be useful for optimisations of other fields 🚀

Reply:
![image](https://github.com/user-attachments/assets/2cd4d1e6-2b88-4ed2-8e81-9fd6a20d504d)
I made sure it wasn't used in our code in case it break something